src/components/TitleBody/TitleBody.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 
-const TitleBody = ({ title, body }) => {
-  
-    return (
-        <>
-            <h2 className="title">{title}</h2>
-            <div className="body" dangerouslySetInnerHTML = {{ __html: body }} />
-        </>
-        )
-    }
+const TitleBody = ({ title, body }) => { 
+  return (
+      <>
+          <h2 className="title-body__title">{title}</h2>
+          <div className="title-body__body" dangerouslySetInnerHTML = {{ __html: body }} />
+      </>
+    )
+}
```

Comment:
🤔 spacing is still off

```suggestion
const TitleBody = ({ title, body }) => { 
  return (
    <>
      <h2 className="title-body__title">{title}</h2>
      <div className="title-body__body" dangerouslySetInnerHTML = {{ __html: body }} />
    </>
  )
}
```